src/seed_data/utils/utilFunctions.ts

```diff
@@ -0,0 +1,51 @@
+import * as faker from 'faker';
+import { LetterGrade } from '../../entity/Enums';
+
+export function getRandomIntInclusive(min: number, max: number) {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max - min + 1) + min); //The maximum is inclusive and the minimum is inclusive
+}
+
+export function toUpperCase(value: String) {
+    const words = value.split(' ');
+    var i: number;
+    for (i = 0; i < words.length; i++) {
+        words[i] = words[i][0].toUpperCase() + words[i].substr(1);
+    }
+
+    return words.join(' ');
+}
+
+export function chooseFrom(options: any[]) {
+    const x = faker.random.number(options.length - 1);
+    return options[x];
+}
+
+export function chooseLetterGrade(percentage: number) {
+    if (percentage >= 93) {
```

Comment:
nit - im letting my imagination go wild here, hear me out:

what if we had some dictionary/map structure like
```
{ 93: LetterGrade.A, 90: LetterGrade.A_MINUS, 87: LetterGrade.B_PLUS, ... }
```
and run through the key set in some loop with an if statement like so
```
if (percentage >= key) {
    return dict[key]
}
```

ofc, either way does the job, so totally up to you

Reply:
👀

Reply:
okay i tried this and the way that _would_ be cleaner just ends up assigning "F" to everything .... obviously there's a way to do it, but I'm not going to put the time in since it's a lot less readable than this. Good thinking though!